oggm/workflow.py

```diff
@@ -283,6 +283,9 @@ def init_glacier_regions(rgidf=None, *, reset=False, force=False,
         if os.path.exists(fpath):
             rmtree(fpath)
 
+    # Read the hash dictionary before we use multiproc
+    utils.get_dl_verify_data('cluster.klima.uni-bremen.de')
```

Comment:
I'm not 100% sure yet, but it's likely I'll remove all caching of the hash file(s) altogether: i.e. each potential verify will trigger a new read. The reason it that I'd like to avoid memory pickling too much data around. What do you think?

Reply:
You can leave it as is for now: we can discuss on slack